src/current/v24.1/known-limitations.md

```diff
@@ -6,185 +6,461 @@ keywords: gin, gin index, gin indexes, inverted index, inverted indexes, acceler
 docs_area: releases
 ---
 
-This page describes newly identified limitations in the CockroachDB {{page.release_info.version}} release as well as unresolved limitations identified in earlier releases.
+{% assign previous_version = site.data.versions | where_exp: "previous_version", "previous_version.major_version == page.version.version" | map: "previous_version" | first %}
 
-## New limitations
+## New limitations in {{ page.version.version }}
 
-### PL/pgSQL Limitations
+The following sections describe newly identified limitations in CockroachDB {{ page.version.version }}.
```

Comment:
```suggestion
The following sections describe newly identified limitations in CockroachDB {{ page.version.version }}.
```

I think during the time when there aren't any new limitations, we should maybe replace this with something saying if any are discovered, they will be listed. WDYT?

Reply:
Done.